### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by P1n3appl3**
Also very slightly modified step 4 of contribution guide

### CODE REVIEW

**Comment by P1n3appl3** [comment]
Idk is this ugly or fine?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Type up your code in whatever text editor you choose, and commit using:
 
 Now, connect your branch to the remote and push your commits:
 
-`git push --set-upstream origin/<name> <name>`
+`git push --set-upstream origin/develop <name>`
```
Comment (CheezBallzPi) [inline CONTRIBUTING.md]: Actually, I screwed up here. It should be:

`git push --set-upstream origin <name>`

Comment (P1n3appl3) [inline CONTRIBUTING.md]: oh yeah otherwise when u push it makes u do more stuff

**Comment by P1n3appl3** [comment]
Should be fixed now with 7727b4f5ea2514a4c4cec29374caf2f6f436f8cb

**Pull request merged**